packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewardsModal/ChallengeRewardsModal.tsx

```diff
@@ -330,6 +336,19 @@ const ChallengeRewardsBody = ({ dismissModal }: BodyProps) => {
     challenge.max_steps > 1 &&
     challenge.challenge_type !== 'aggregate'
 
+  /* Filter for only claimable challenges */
+  const getClaimableChallengeSpecifiers = useCallback(
```

Comment:
Curious about using `useCallback` here. Was the intent to memoize the result? As it is, it will still be computed each time it's called. And if those values don't change often, it might make more sense to do `useMemo`?
Or, if this is just meant to be a helper for the `onClaimRewardClicked` below, maybe move it outside of the component and pass in both `specifiers` and `undisbursedChallenges` so we aren't running through the `useCallback` check on every render when we only need the logic during the click handler?

Reply:
ty great callout